lib/tokenize.cpp

```diff
@@ -3002,7 +3002,7 @@ static bool setVarIdParseDeclaration(const Token **tok, const std::map<std::stri
             bracket = true; // Skip: Seems to be valid pointer to array or function pointer
         } else if (tok2->str() == "::") {
             singleNameCount = 0;
-        } else if (tok2->str() != "*" && tok2->str() != "::") {
+        } else if (tok2->str() != "*" && tok2->str() != "::" && tok2->str() != "...") {
```

Comment:
Will thie be executed for C code as well?

Reply:
Yes but it is already checking for "::" in a few places without checking the language.  Should they all be fixed?

Reply:
I believe that we don't need special handling of C code for `::`.. valid C code does not contain that operator.

But `...` can be used in C code. However if there is valid C code that contains `...` I would assume that should be handled the same way in C++ mode... so this either does not break C or it breaks both C and C++ parsing.